docs/build-and-run-a-docker-image.md

```diff
@@ -4,6 +4,10 @@
 
 - Install [Docker](https://www.docker.com)
 - Already have a PostgreSQL 9.6 database running
+- Follow the instructions of the main [README](https://github.com/opendatakit/aggregate), including:
```

Comment:
Do you think we could link to the [Getting the code](https://github.com/opendatakit/aggregate#getting-the-code) section directly?

```markdown
[Getting the code](https://github.com/opendatakit/aggregate#getting-the-code)
```

Reply:
Good idea, have updated my PR!